pkgs/by-name/ru/rustdesk-flutter/package.nix

```diff
@@ -206,6 +207,12 @@ flutter319.buildFlutterApplication rec {
     cp ../res/scalable.svg $out/share/icons/hicolor/scalable/apps/rustdesk.svg
   '';
 
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "${pipewire}/lib/gstreamer-1.0"
```

Comment:
This should be handled by wrapGAppsHook automatically if you add pipewire to buildInputs?